controllers/activemqartemis_controller_test.go

```diff
@@ -6309,6 +6307,151 @@ var _ = Describe("artemis controller", func() {
 		}, timeout, interval).Should(BeTrue())
 	})
 
+	It("ordinal broker properties", func() {
+		ctx := context.Background()
+		crd := generateArtemisSpec(defaultNamespace)
+		crd.Spec.DeploymentPlan.Size = 2
+		crd.Spec.BrokerProperties = []string{
+			"name=BROKER_ORDINAL_0",
```

Comment:
Is this 'name' only overrides pod-0 or all the broker-n (except broker-1)? I think some doc need here to explain the behavior. It looks like if you scale up to 3 the 3rd doesn't use this name. (I saw some log like File /amq/extra/secrets/artemis-broker-props/broker-2 does not exist), maybe it's a bug.

Reply:
properties without a prefix end up in <mount point>/broker.properties in the normal way. Properties with a prefix end up in <mount-point>/broker-N/broker-N.broker.properties and each broker looks in mount-point and mount-point/broker-${STATEFUL_SET_ORDINAL} - such that it only picks up the prefixed properties that match.

let me check with a test for scale up, I think the primary use case is static pods, however any pod that does not have extra prefixed props, will get the normal config.

Reply:
and I will add some doc, in the help/operator.md - there is currently no brokerProperties section, I will fix that. thanks

Reply:
it is better now